backends/json/json.cc

```diff
@@ -149,10 +182,10 @@ struct JsonWriter
 				}
 			}
 			f << stringf("          \"parameters\": {");
-			write_parameters(c->parameters);
+			write_attributes(c->parameters);
```

Comment:
Shouldn't this be write_parameters?

Reply:
Need to explore that. Cell parameters are handled differently since they have actual values assigned. For now I kept the original behavior, just changed the function name.

Reply:
There is no need for changing behavior of writing parameters for cell instances.